source/content/advanced-global-cdn.md

```diff
@@ -1,75 +1,127 @@
 
-## Features
+### IP Allowlisting & Blocklisting
+
+ Granular access control down to the IP address level helps protect intellectual property and secure development environments.
+
+- Blocklist - prevent users with specific IPs or in IP ranges from accessing your site.
 
-- **Full-page & asset caching**
+- Allowlist - only allow users with specific IPs or in IP ranges to access your site, blocking everyone else.
 
-  Unlike most CDNs that only handle images and other static assets, full-page technology lets you cache entire pages and assets at the edge.
-- **Layer 3 & 4 DDOS protection & mitigation**
-- **SOC2 compliance**
-- **IP allowlisting & blocklisting**
-  - **Blocklist** - prevent users with specific IPs or in IP ranges from accessing your site.
-  - **Allowlist** - only allow users with specific IPs or in IP ranges to access your site, blocking everyone else.
-- **Token authentication / JWT (JSON Web Token)**
+### Token Authentication / JWT (JSON Web Token)
 
   Use the AGCDN Edge to generate JSON Web Tokens<Popover title="JSON Web Tokens" content="A JSON Web Tokens is an Internet standard for creating compact, encrypted JSON-based access tokens that assert some number of claims, such as 'logged in as admin'." /> to help build your custom API.
-- **Custom error & maintenance page rules**
-- **Geolocation based edge control**
 
-  Take action on the edge based on the physical location of the user (eg. blocking or redirecting based on country) or send back location information to your application for more custom control.
-- **Device detection and edge control**
-- **Persistent Uptime**
+### Custom Error & Maintenance Page Rules
+
+ Response code messages enable you to create custom error pages (e.g. 404 and 503 errors) for a tailored end-user experience.
+
+### Geolocation-based Actions
+
+  Enable actions based on the physical location of your user (eg. blocking or redirecting based on country) or send location information to your application for more custom control.
 
-  Adjusts the serve stale content setting.
-- **Full CDN edge logs**
+### Device Detection and Edge Control
+
+ [Device detection](https://docs.fastly.com/en/guides/delivering-different-content-to-different-devices) can drive differentiated delivery strategies at the edge of the network, to address issues that impact mobile user experience.
+
+### Persistent Uptime
+
+  Adjusts the "serve stale content" setting to make content available from the edge even if the site is unavailable.
+
+### Full CDN Edge Logs
 
   See every request to your website, whether or not the content was cached. Full CDN edge logs combine the server logs with logs from the edge to provide a complete picture of your traffic.
```

Comment:
This isn't technically accurate - we're not combining data; rather, the edge logs take the place of incomplete server logs (incomplete because the server logs only record requests that couldn't be fulfilled from GCDN's cache).